### PR TITLE
Apply post-review css changes

### PR DESCRIPTION
**Pull request by cella**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -706,6 +743,7 @@ <h1 style="font-size: 48px;">My headline</h1>
 						### Class Selector
 						- The class selector selects elements with a specific class attribute
 						- Use a period (`.`) character, followed by the name of the class
+						- Can be appear multiple times within a page, unlike id selectors which only appear once
```
Comment (alexelash) [inline index.html]: Tiny typo! Just remove "be"

File: index.html [hunk 2]
```diff
@@ -1121,14 +1183,21 @@ <h1 style="font-size: 48px;">My headline</h1>
...
 						---
 
-						### Exercise
+						### Borders
 
-						- Try adding the following properties to your project divs
-							- Padding
-							- Margin
-							- Border
+						- You can set the border to have various styles
```
Comment (alexelash) [inline index.html]: This is small--and not vital, so feel free to ignore :) --but for slides with only a single bullet, we can probably remove the list style entirely to simplify the content on the page! I only noticed on these several slides.

**Review by alexelash — APPROVED** [review]
Looks great, Cella! I made a couple of comments but they're really small.

File: index.html (hunk 1 above)
Comment (cella) [reply index.html]: done!

**Comment by cella** [comment]
fixed those up!

**Pull request merged**